packages/fxa-settings/src/components/LinkDamaged/index.tsx

```diff
@@ -0,0 +1,55 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { FtlMsg } from 'fxa-react/lib/utils';
+
+type LinkDamagedProps = {
+  linkType: LinkType;
+};
+
+type LinkType = 'reset-password' | 'signin';
```

Comment:
This is good as-is, might be even better with enums! (just a nit pick)

Reply:
oh nice makes sense

Reply:
I'm leaving this as-is just bc it's a bit of a time crunch getting all the tickets in, but will keep it in mind!

Reply:
I'd love to discuss this further - I don't know a lot about typescript but had seem videos etc pushing against enums bc of unexpected behavior at runtime! Do we already have a defined opinion on them as a team?